.github/workflows/build.yml

```diff
@@ -7,7 +7,7 @@ on:
       - '**.md'
       - '**.txt'
   schedule:
-    - cron: '20 20 * * *'  # 8:20pm everyday
+    - cron: '0 7 * * *'  # 7 am everyday
```

Comment:
Question on this change... this is 7:00am UTC.

I checked the last several days of https://quay.io/repository/fedora-ostree-desktops/silverblue?tab=tags (also looked at base, kinoite, sericea, vauxite) and it seems they are generally done building by 4am UTC...

So it seems this is a good change as it leaves a few hours of buffer for those to complete. 

But I wanted to confirm as the original 8:20pm time was also planned to run an hour or so after a daily build. What changed? Was the old time based on the cgwalters images?

Reply:
This was answered in Discord. New time is as I expected. Looks good to me.